variants/MALYANM200_F103CB/PinNamesVar.h

```diff
@@ -0,0 +1,25 @@
+    /* SYS_WKUP */
```

Comment:
Could be update to add USB pin name
https://github.com/stm32duino/Arduino_Tools/blob/master/src/genpinmap/Arduino/STM32F103C(8-B)Tx/PinNamesVar.h

Reply:
Done, thank you.

Reply:
Thanks